data-serving/data-service/schemas/cases.schema.json

```diff
@@ -9,6 +9,21 @@
       "__v": {
         "bsonType": "int"
       },
+      "caseReference": {
+        "bsonType": "object",
+        "additionalProperties": false,
+        "properties": {
+          "_id": {
+            "bsonType": "objectId"
+          },
+          "dataSourceId": {
```

Comment:
Will this be a foreign key to the _id from the sources collection? If so, do we need to keep the "sources" field in this schema? (Not suggesting deletion in this PR, should be it's own if necessary)

Reply:
Yep, certainly a good question. This is related to my comment in the PR description re: `Sources`. Let's catch up with Kelly on this next week -- I'm not sure what her plans were for `Sources` now that the spec is finalized.